{"uid":"7ec43f4589283c4","name":"User cannot login with incorrect loginname","fullName":"cypress/e2e/authorization.cy.js#Negative authorization test suite User cannot login with incorrect loginname","historyId":"08c41ac3c07a36320ccb3091f6366988:d41d8cd98f00b204e9800998ecf8427e","time":{"start":1720642340080,"stop":1720642345681,"duration":5601},"status":"broken","statusMessage":"Timed out after `4000ms`. The `done()` callback was never invoked!","statusTrace":"CypressError: Timed out after `4000ms`. The `done()` callback was never invoked!\n    at cypressErr (https://automationteststore.com/__cypress/runner/cypress_runner.js:75613:18)\n    at Object.errByPath (https://automationteststore.com/__cypress/runner/cypress_runner.js:75668:10)\n    at https://automationteststore.com/__cypress/runner/cypress_runner.js:145547:72","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"Timed out after `4000ms`. The `done()` callback was never invoked!","statusTrace":"CypressError: Timed out after `4000ms`. The `done()` callback was never invoked!\n    at cypressErr (https://automationteststore.com/__cypress/runner/cypress_runner.js:75613:18)\n    at Object.errByPath (https://automationteststore.com/__cypress/runner/cypress_runner.js:75668:10)\n    at https://automationteststore.com/__cypress/runner/cypress_runner.js:145547:72","steps":[],"attachments":[{"uid":"3d5ae5f010e961f1","name":"Screenshot","source":"3d5ae5f010e961f1.png","type":"image/png","size":121907},{"uid":"ba7795660ac5e3ca","name":"Video","source":"ba7795660ac5e3ca.mp4","type":"video/mp4","size":3102160}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"language","value":"javascript"},{"name":"framework","value":"cypress"},{"name":"parentSuite","value":"Negative authorization test suite"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":2,"skipped":0,"passed":0,"unknown":0,"total":2},"items":[{"uid":"63502345e3c65427","reportUrl":"https://SanitarskiyDima.github.io/G8-test-store/13//#testresult/63502345e3c65427","status":"broken","statusDetails":"Timed out after `4000ms`. The `done()` callback was never invoked!","time":{"start":1720642278802,"stop":1720642284520,"duration":5718}}]},"tags":[]},"source":"7ec43f4589283c4.json","parameterValues":[]}